platform-sdk/swirlds-jasperdb/build.gradle.kts

```diff
@@ -15,26 +15,27 @@
  */
 
 plugins {
-  id("com.swirlds.platform.conventions")
-  id("com.swirlds.platform.library")
-  id("com.swirlds.platform.maven-publish")
-  id("com.swirlds.platform.benchmark-conventions")
+    id("com.swirlds.platform.conventions")
+    id("com.swirlds.platform.library")
+    id("com.swirlds.platform.maven-publish")
+    id("com.swirlds.platform.benchmark-conventions")
 }
```

Comment:
Are these formatting changes to this file intentional?

Reply:
I thought it's done by Spotless. Let me revert it and see if the build fails

Reply:
yep. it was spotless